packages/vaadin-grid-pro/src/vaadin-grid-pro-edit-column.js

```diff
@@ -269,19 +271,9 @@ class GridProEditColumnElement extends GridColumnElement {
    * @protected
    */
   _stopCellEdit(cell, model) {
-    const editor = this._getEditorComponent(cell);
-    let shouldResize = true;
-    if (editor) {
-      editor.removeEventListener('focusout', this._grid.__boundEditorFocusOut);
```

Comment:
Since the `_clearCellContent` method clears the cell content, it should remove the editor component as well, so that there is no need to remove these event listeners manually.